src/stx/common/trex_rx_timesync.cpp

```diff
@@ -0,0 +1,107 @@
+#include "trex_rx_timesync.h"
+
+#include <trex_global.h>
+
+/**************************************
+ * RXTimesync
+ *************************************/
+// RXTimesync::RXTimesync() {}
```

Comment:
cleanup

Reply:
done in bd1aed19c572e9eb48e42a37603797844290c27c